Lesson23/Mesut/map.js

```diff
@@ -360,6 +373,14 @@ const booksExercise9 = [
 	},
 ];
 
+const booksExercise9Result = booksExercise9.map(book=>book.title+ ", "+book.author)
+console.log(booksExercise9Result)
+
+
 // Exercise 10:
 // Given an array of prices, add a currency symbol to each price using .map().
 const pricesExercise10 = [1999, 899, 1499, 299, 399, 999, 499, 649, 99, 129];
+
+const pricesExercise10Lira = pricesExercise10.map(currency=>currency+"₺")
```

Comment:
Since you're given an array of prices, a single item can be called "price". Calling it "currency" is misleading.
```suggestion
const pricesExercise10Lira = pricesExercise10.map(price => price + "₺");
```